data/remnant/remnant missions.txt

```diff
@@ -1054,12 +1066,211 @@ mission "Remnant: Deep Surveillance"
+			choice
+				`	"They know where the Remnant are in the Ember Waste."`
+					goto lcoation
+				`	"They have scans of a few Remnant ships."`
+					goto scans
+				`	"They have samples of some Remnant technology."`
+					goto tech
+			
+			label surveillance
+			choice
+				`	"They know where the Remnant are in the Ember Waste."`
+					goto location
+				`	"They have scans of a few Remnant ships."`
+					goto scans
+				`	"They have samples of some Remnant technology."`
+					goto tech
+				`	"They are actively surveilling Remnant worlds."`
+					goto surveilled
+			
+			label unknown
+			choice
+				`	"They don't know about the Remnant, at least not yet."`
+				`	"They have only discovered the void sprites."`
+					goto sprites
+			
+			`	Torza seems to calm down greatly after hearing this. "This is most fortunate. We will need to work to ensure that this remains the case."`
+				goto end
+			
+			label sprites
+			`	Torza seems to calm down greatly after hearing this. "Then may they not anger the Archon with their curiosity as we have."`
+				goto end
+			
+			label location
+			`	Torza pauses for a long while. "I assume it only makes sense that they would know of us after discovering the Ember Waste."`
+				goto end
+			
+			label scans
+			`	Torza pauses for a long while. "These scientists knowing of our whereabouts is much more of a concern than them having scans of our ships."`
+				goto end
+			
+			label tech
+			`	"And you were the one who brought it to them," Torza responds.`
+			branch reciprocal 
+				has "Remnant: Tech Retrieval: done"
+			choice
+				`	"Yes, but what I do with your technology after buying it is none of your concern."`
+					goto concern
+				`	"No, they obtain it by other means."`
+					goto other
+			
+			label reciprocal
+			choice
+				`	"Yes, but it is no different than me bringing technology from the Republic to you."`
+				`	"No, they obtained it by other means."`
+					goto other
+			
+			`	Torza scowls in response to this, but doesn't say anything.`
+				goto end
+			
+			label concern
+			`	"What you do with our technology is much of a concern to us!" Torza yells.`
+				goto end
+			
+			label other
+			apply
+				set "remnant: lied to torza: technology"
+			`	Torza grits his teeth. "There are no reports of missing ships or outfits, and the only visitor capable of delivering our technology is you."`
+				goto end
 			
-			`	`
+			label surveilled
+			apply
+				set "remnant: told about surveillance"
+			`	"How are they doing this?" Torza asks.`
+			choice
+				`	"Surveillance devices have been placed in each of your spaceports."`
+				`	"I don't know, I only know that they are."`
+					goto secret
+			
+			apply
+				set "remnant: told about cubes"
+			`	Torza punches a wall, then relaxes. "Thank you for telling us this. We will search for and remove these devices immediately."`
+				goto end
+			
+			label secret
+			`	"Whatever methods they are using, we will discover them in due time," Torza says.`
```

Comment:
I find it very incongruous that the Remnant would not know about the surveillance devices:
a) They have very advanced scanners and it is already established that they regularly do detailed scans of <first>'s ship (for instance, the Broken JD missions)
b) Virtually all Remnant know of <first>, and they stand out in Remnant society as a stranger. There is no chance that <first> "blends into the crowd"
c) Remnant have a strong familiarity with cloaking devices, and have mentioned that smaller things are easier to cloak than larger things. Personal cloaks for surveillance operatives or devices is not unlikely, but a surveillance starling (when <first> strolls out of town) is virtually guaranteed.
d) Remnant are hyper-security conscious. While they may let <first> buy their stuff and be starting to trust them more, it is highly unlikely that they would let them wander off without surveillance. On the one hand, <first> is still a stranger. One that appears trustworthy, but a stranger nontheless. On the other hand, <first> is a valuable asset for gaining close-up knowledge of the rest of the galaxy that they hithertoo have only seen from a distance. It'd be a shame if such a valuable asset was put out of commission by getting lost in a desert or falling off a cliff or into some piece of machinery or something. 

As such, I consider this surprise and outburt of Torza's is either erroneous, or an act intending to deceive the player.

Reply:
I actually thought of doing that whole “pull out the cubes” thing here. 

As far as stripping of the license and hostility goes, that’s something I also thought of doing here, but didn’t want to go that far without a go ahead from you.

Reply:
Go for it. I am *all* on board with having Torza confront the player about the cubes, and possibly stripping license(s) depending on how the conversation goes.